Source/FAQView.swift

```diff
@@ -360,67 +400,76 @@ class FAQViewCell: UITableViewCell {
   func configure(configuration: FAQConfiguration) {
     self.backgroundColor = configuration.cellBackgroundColor
     self.questionLabel.textColor = configuration.questionTextColor
-    self.answerLabel.textColor = configuration.answerTextColor
+    self.answerTextView.textColor = configuration.answerTextColor
     self.questionLabel.font = configuration.questionTextFont
-    self.answerLabel.font = configuration.answerTextFont
+    self.answerTextView.font = configuration.answerTextFont
+    if let dataDetectorTypes = configuration.dataDetectorTypes {
+      self.answerTextView.dataDetectorTypes = dataDetectorTypes
+    }
+    if let tintColor = configuration.tintColor {
+      self.answerTextView.tintColor = tintColor
+    }
   }
   
   func expand(withAnswer answer: String, animated: Bool) {
-    self.answerLabel.text = answer
+    answerTextView.text = answer
+    expand(animated: animated)
+  }
+  
+  func expand(withAttributedAnswer attributedAnswer: NSAttributedString, animated: Bool) {
+    answerTextView.attributedText = attributedAnswer
+    expand(animated: animated)
+  }
+  
+  func expand(animated: Bool) {
+    answerTextView.isHidden = false
     if animated {
-      self.answerLabel.alpha = 0
+      answerTextView.alpha = 0
       UIView.animate(withDuration: 0.5, animations: {
-        self.answerLabel.alpha = 1
+        self.answerTextView.alpha = 1
       })
     }
-    self.answerLabelBottom.constant = 20
-    self.update(arrow: .Up, animated: animated)
+    answerTextViewBottom.constant = 20
+    update(arrow: .up, animated: animated)
   }
   
-  
   func collapse(animated: Bool) {
-    self.answerLabel.text = ""
-    self.answerLabelBottom.constant = 0
-    self.update(arrow: .Down, animated: animated)
+    answerTextView.text = ""
+    answerTextView.isHidden = true
+    answerTextViewBottom.constant = -20
+    update(arrow: .down, animated: animated)
   }
   
   func update(arrow: Arrow, animated: Bool) {
     switch arrow {
-    case .Up:
-      if animated {
-        // Change direction from down to up with animation
-        self.indicatorImageView.rotate(withAngle: CGFloat(0), animated: false)
-        self.indicatorImageView.rotate(withAngle: CGFloat(M_PI), animated: true)
-      } else {
-        // Change direction from down to up without animation
-        self.indicatorImageView.rotate(withAngle: CGFloat(M_PI), animated: false)
-      }
-    case .Down:
-      if animated {
-        // Change direction from up to down with animation
-        self.indicatorImageView.rotate(withAngle: CGFloat(M_PI), animated: false)
-        self.indicatorImageView.rotate(withAngle: CGFloat(0), animated: true)
-      }
+    case .up:
```

Comment:
Hey, can you change this part to whatever was there earlier? because now it won't animate properly. We have to rotate the arrow image twice to show the animation.